src/Tools/PerfDiff/Program.cs

```diff
@@ -18,48 +15,26 @@ namespace PerfDiff
     internal sealed class Program
     {
         internal const int UnhandledExceptionExitCode = 1;
-        private static ParseResult? s_parseResult;
 
-        private static async Task<int> Main(string[] args)
-        {
-            var rootCommand = DiffCommand.CreateCommandLineOptions();
-            rootCommand.Handler = CommandHandler.Create(new DiffCommand.Handler(RunAsync));
-
-            // Parse the incoming args so we can give warnings when deprecated options are used.
```

Comment:
I was not able to find any other usage of `Program.RunAsync` so I've assumed that it's most likely some leftover and removed it.

FWIW if there will be any parsing errors detected, the `Parse(args).InvokeAsync()` call is going to get them printed to output and also return error code.